python/cdp_san_jose_backend/clean_database.py

```diff
@@ -0,0 +1,48 @@
+from cdp_backend.database import models as db_models
+import fireo
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.firestore import Client
+
+# Connect to the database
+fireo.connection(client=Client(
+    project="cdp-san-jose-5d9db455",
+    credentials=AnonymousCredentials()
+))
+
+# Read from the database
+sessions = db_models.Session.collection.fetch(1000)
+
+i=0
+for session in sessions:
+    if session.session_datetime.year == 2020 and session.session_datetime.month==1 and session.session_datetime.day==7:
+        print("Session_time=" + str(session.session_datetime) + " ID=" + str(session.id) + " key=" +str(session.key) +" _update_time=" + str(session._update_time))
+    i=i+1
+
+print("----")
+
+dictionary = dict()
+sessions_for_delete = []
+sessions = db_models.Session.collection.fetch(1000)
```

Comment:
I guess for this specific case, fetching just 1000 records is probably enough(?) to find all the duplicates... But as a standalone script, this simply cannot be true. Fetching in chunks is a good idea but in a vacuum there is no guarantee there is more/less records in the database.

Reply:
Agree in general @dphoria but fetching 1000 is fine. I think it turns out to be KBs and we currently only have ~300 sessions in the database iirc. So this will get everything.

Reply:
Basically. I am fine with merging this as is.

Reply:
Thanks!